hw3/03-form.js

```diff
@@ -0,0 +1,32 @@
+const http = require('http');
+const fs = require('fs');
+const port = process.env.PORT || 5001;
+
+const server = http.createServer((req, res) => {
+    let url = new URL(req.url, `http://${req.headers.host}`);
+    res.setHeader('Content-Type', 'text/html');
+
+    if (req.url === '/') {
+        res.end(fs.readFileSync(__dirname + '/03-form.html'));
+    } else if (url.pathname === '/submitForm') {
+        const parameters = url.searchParams;
+        let text = '';
+        parameters.forEach((value, key) => {
+            if (key == 'Newsletter') {
```

Comment:
remember to use triple equals for comparisons unless you really intend for coercions between types